cpp/benchmarks/io/orc/orc_reader_benchmark.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <benchmark/benchmark.h>
+
+#include <benchmarks/common/generate_benchmark_input.hpp>
+#include <benchmarks/fixture/benchmark_fixture.hpp>
+#include <benchmarks/io/cuio_benchmarks_common.hpp>
+#include <benchmarks/synchronization/synchronization.hpp>
+
+#include <cudf/io/functions.hpp>
+
+// to enable, run cmake with -DBUILD_BENCHMARKS=ON
+
+constexpr int64_t data_size = 512 << 20;  // 512 MB
+
+namespace cudf_io = cudf::io;
+
+template <typename T>
+class OrcWrite : public cudf::benchmark {
+};
+
+template <typename T>
+void ORC_write(benchmark::State& state)
```

Comment:
Should this be ORC_read?

```suggestion
void ORC_read(benchmark::State& state)
```